README.md

```diff
@@ -30,8 +30,9 @@ $container['view'] = function ($c) {
     ]);
     
     // Instantiate and add Slim specific extension
-    $basePath = rtrim(str_ireplace('index.php', '', $c['request']->getUri()->getBasePath()), '/');
-    $view->addExtension(new \Slim\Views\TwigExtension($c['router'], $basePath));
+    $router = $c->get('router');
+    $uri = Slim\Http\Uri::createFromEnvironment(new \Slim\Http\Environment($_SERVER)):
```

Comment:
Isn't the environment already registered in the container?

Reply:
I'm trying to discourage use of Environment, request or response in the container as they won't be there in 4.

Reply:
note - there is a colon and not a semicolon at the end of line 34 in the readme file

Reply:
oh noes!

Reply:
Doh! Fixed & push.